src/store/stub/store.py

```diff
@@ -24,16 +31,16 @@ def setup(self):
             self.publishers = json.load(f)
 
         with open(Path(content_dir / "topics.json").absolute()) as f:
-            self.topics = json.load(f)
+            self.themes = json.load(f)
```

Comment:
can you change themes to topics please, I had to correct that earlier on another branch so the history is well confusing, but its topics that we want.